haystack/management/commands/build_solr_schema.py

```diff
@@ -42,17 +99,17 @@ def build_context(self, using):
         content_field_name, fields = backend.build_schema(
             connections[using].get_unified_index().all_searchfields()
         )
-        return Context({
+        return {
             'content_field_name': content_field_name,
             'fields': fields,
             'default_operator': constants.DEFAULT_OPERATOR,
             'ID': constants.ID,
             'DJANGO_CT': constants.DJANGO_CT,
             'DJANGO_ID': constants.DJANGO_ID,
-        })
+        }
 
-    def build_template(self, using):
-        t = loader.get_template('search_configuration/solr.xml')
+    def build_template(self, using, tfile=schema_template_loc):
```

Comment:
Since this is a keyword argument it should be something like `template_filename` which is a little more obvious when someone looks at the help, uses autocomplete in an editor, etc.